README.md

```diff
@@ -83,6 +83,11 @@ This library is intended to provide a quicker and easier way to get started usin
 * `uint8_t getAddress(void)`<br>
   Returns the current I&sup2;C address.
 
+* `bool begin(void)`<br>
+  Iniitializes the sensor to use Wire as the default port. 
+* `bool begin(&Wire1)`<br>
```

Comment:
`bool begin(TwoWire *theWire)`

It should show the the type and the name of the parameter.

If we changed the interface of begin to be `begin(TwoWire &theWire) and used the address operator (&) inside begin, the users of this library would not have to understand what & does.

Reply:
Hi Ryan.  Made all other changes.  Not sure how I deleted the history in the Readme though. Anyway, this is the only one that confuses me - pointers and addresses always did that anyway.  Are you saying that it should be:
begin(TwoWire &theWire) { _wire = &theWire; ... } or _wire = *theWIre?

What is the form of the calling function in the .ino - begin(Wire) ?

Reply:
The first one `begin(TwoWire &theWire) { wire = &theWire; ... }` then yes, you call it with `begin(Wire1)`. We do it in our Maestro library https://github.com/pololu/maestro-arduino/blob/master/PololuMaestro.cpp#L5 with `stream`.

Reply:
Changes made and pushed update.  Thanks for the lesson.  Funny I have a couple of Maestro (which I love by the way) that I use for my robotic arm and I never noticed that one.  Tested on a Teensy 3.5 with Wire only.  I will have to solder up another sensor if you want me to test different Wire configure.

Thanks
Mike